spec/spec_helper.rb

```diff
@@ -10,6 +10,7 @@
 require 'rspec/rails'
 require 'rack_session_access/capybara'
 require 'umts_custom_matchers'
+require 'paper_trail/frameworks/rspec'
```

Comment:
While we're here (I was making sure we actually have some tests for the "undo" feature).